### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by Cheridan**
-Fixes magic wands having electronic firing pins, now have FANCY CRYSTAL SHARDS
-Fixes grammer issue.
-Fixes code not compiling due to outdated proc

### CODE REVIEW

File: code/modules/projectiles/gun.dm [hunk 1]
```diff
@@ -46,7 +46,7 @@
 /obj/item/weapon/gun/examine(mob/user)
 	..()
 	if(pin)
-		user << "It has a [pin] installed."
+		user << "It has [pin] installed."
```
Comment (Delimusca) [inline code/modules/projectiles/gun.dm]: wouldn't this say "It has the electronic firing pin installed" , that looks kind of weird
why not put \an and capitalize the firing pin names?

Comment (Cheridan) [inline code/modules/projectiles/gun.dm]: Capitalized item names aren't standardized, and would causes issues elsewhere as the game would consider it a Proper Noun.

Comment (Delimusca) [inline code/modules/projectiles/gun.dm]: it just hit me how exponentially worse that would make everything else. nevermind then, better than usual byond grammar anyway.

Comment (Miauw62) [inline code/modules/projectiles/gun.dm]: just make this "It has a [pin.name] installed.", that would fix most of it.

**Pull request merged**